src/infra/docs/docs-rs/README.md

```diff
@@ -0,0 +1,14 @@
+# docs.rs
```

Comment:
Could you add a small summary about what this section is documenting. It doesn't have to be long, one or two lines is enough.